packages/i18nify-js/src/modules/currency/data/currencyConfig.json

```diff
@@ -32,12 +32,12 @@
   "XCD": {
     "name": "East Caribbean Dollar",
     "minor_unit": "2",
-    "symbol": "$"
+    "symbol": "EC$"
```

Comment:
This file is generated at build time right ?

Reply:
Yes.